### DISCUSSION TITLE
Switching between pages in navigation view

### DISCUSSION THREAD

**Discussion by recursivevoidptr** [Kinnara/ModernWpf#459] [General]
I'm completely new to .NET and I created a button inside a page. Whenever the button is pressed it should change the page in a specific page. In UWP, i use this.Frame.Navigate
Is there a way to do it in ModernWpf?

**Comment by satpal123** [comment]
You can do it using the code below. On button click, you can switch to a different page.

https://github.com/Kinnara/ModernWpf/blob/2cd256ffe38326d1d4822b5187e2e673fc049990/ModernWpf.SampleApp/ControlPages/NavigationViewPage.xaml.cs#L68
